packages/worker/lib/workflows.ts

```diff
@@ -10,7 +10,8 @@ const { routeSync, scheduleAndRouteSync, runAction } = proxyActivities<typeof ac
     retry: {
         initialInterval: '5m',
         maximumAttempts: 3
-    }
+    },
+    heartbeatTimeout: '10m'
```

Comment:
I will have a timeout strictly greater than the heartbeat interval, just to avoid some weird scheduling mismatch. Even 15mins would be fine